src/components/MusicPage/MusicPage.js

```diff
@@ -9,53 +9,79 @@ import {
   ShowFormatCooldown,
 } from '../WorkoutCard/ShowFormatFromSections'
 import { selectColor, selectColorByInterval } from '../../services/selectColor'
+import Header from '../Header'
 
 export default function MusicPage({ location, onCreatePlaylist }) {
   const { name, warmup, intervalsT, cooldown } = location.workout
 
   return (
-    <Grid>
-      <Heading> Select your music:</Heading>
-
-      <MusicForm
-        workout={location.workout}
-        onCreatePlaylist={onCreatePlaylist}
-      />
-
-      <CardGrid>
-        <Title>
-          Your selected workout:
-          <br /> {name}
-        </Title>
-
-        <Wrapper>
-          <div>
-            <WarmupSection background={selectColor(warmup.powerHigh)}>
-              <ShowFormatWarmup warmup={warmup} />
-            </WarmupSection>
-            <IntervalsTSection
-              background={selectColorByInterval(intervalsT.onPower)}
-            >
-              <ShowFormatIntervalsT intervalsT={intervalsT} />
-            </IntervalsTSection>
-            <CooldownSection background={selectColor(cooldown.powerHigh)}>
-              <ShowFormatCooldown cooldown={cooldown} />
-            </CooldownSection>
-          </div>
-        </Wrapper>
-      </CardGrid>
-
-      <BackButton as={NavLink} to={{ pathname: '/' }}>
-        <Icon glyph="back" size={24} />
-        Back
-      </BackButton>
-    </Grid>
+    <div>
+      <HeaderStyled>Music</HeaderStyled>
+
+      <Grid>
+        <Heading> Select your music:</Heading>
+
+        <MusicForm
+          workout={location.workout}
+          onCreatePlaylist={onCreatePlaylist}
+        />
+
+        <CardGrid>
+          <Title>
+            Your selected workout:
+            <br /> {name}
+          </Title>
+
+          <Wrapper>
```

Comment:
Wrapper for what?
 maybe it's ok, but it's easier if you have a specific nameWrapper

Reply:
ok. I´ll be mor specific. Thanks!